src/utilities/spacing/spacing.scss

```diff
@@ -41,11 +47,17 @@
     }
   }
 
+  /**
+    * Ideally we could use the *-inline shorthand, but it is not as well
+    * supported in browsers.
+    */
```

Comment:
Nit: We usually align the _first_ asterisks (not the second), like jsdoc.

Reply:
I tried that and the linter changed it. I'll try again.

Reply:
No idea why, but this time I reverted it to the way I had it and the linter didn't complain, so updated.

Reply:
@tylersticka, whenever you have a moment ^